largest-contentful-paint/image-upscaling.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE HTML>
+<meta charset=utf-8>
+<title>Largest Contentful Paint: largest image is reported.</title>
+<body>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/largest-contentful-paint-helpers.js"></script>
+<script src="/common/utils.js"></script>
+<script>
+    setup(() =>
+        assert_implements(window.LargestContentfulPaint, "LargestContentfulPaint is not implemented"));
+
+    const imageURL = `${window.location.origin}/images/blue.png`;
+    async function load_image_and_get_lcp_size(t, imageStyle = {}, containerStyle = {}) {
+        const popup = window.open('about:blank');
+        t.add_cleanup(() => popup.close());
+        const image = popup.document.createElement('img');
+        image.src = imageURL;
+        await image.decode();
+        const naturalSize = image.width * image.height;
+        const container = popup.document.createElement('div');
+        container.appendChild(image);
+        const applyStyle = (el, style = {}) =>
+            Object.entries(style).forEach(([k, v]) => el.style.setProperty(k, v));
+
+        applyStyle(image, imageStyle);
+        applyStyle(container, containerStyle);
+        image.id = token();
+        const entryReported = new Promise(resolve => new popup.PerformanceObserver(entryList => {
+            entryList.getEntries().forEach(entry => {
+                if (entry.id === image.id)
+                    resolve(entry.size);
+            });
+        }).observe({type: 'largest-contentful-paint'}));
+        popup.document.body.appendChild(container);
+        const timeout = new Promise(resolve => t.step_timeout(() => resolve('not reported'), 1000));
+        return {lcpSize: (await Promise.any([entryReported, timeout])), naturalSize};
+    }
+
+    const load_background_image_and_get_lcp_size = (t, style) =>
+        load_image_and_get_lcp_size(t, {display: 'none'},
```

Comment:
Why display:none?

Reply:
To have only the div with the background and not the image.